Documentation/PackedAssetBuilds.md

```diff
@@ -1,64 +1,121 @@
 
-![WWise installer options](Media/wwise_installer_options.png)
+![Wwise installer options](Media/Wwise_installer_options.png)
 
 Inside the installer select the version to use.  Install version **2021.1.11.7933**, select both Authoring and SDK, Microsoft platform.
 
-![WWise version selection](Media/wwise_installer_version_selection.png)
+![Wwise version selection](Media/Wwise_installer_version_selection.png)
 
 > REBOOT (or logout / login). Otherwise, the environment settings won't get picked up for any builds in Visual Studio. They will only apply to command-line builds, and only for any command-line windows that have been opened after the installer finishes.
 
-### Build profile build and process assets
+### Build o3de-multiplayersample and the engine (non-monolithic)
+
+Build a regular profile build of the game as per the [README.md](../README.md) in an engine-centric way. Be sure to build from the source engine and not the installer. Here is an example:
+
+Clone the engine from source.
 
-Build and run MPS as per the [README.md](../README.md) and ensure all assets are built.
+```shell
+C:\git> git clone https://github.com/o3de/o3de
+```
 
-### Test the profile build
+Navigate to `C:\git\o3de`.
+Create a build folder for non-monolithic build of the engine, for example `C:\git\o3de\build\non_mono`.
 
-* Open the game in editor
-    * load `NewStarBase` level
-    * Verify that game can launch and connect to local server from editor
-* Validate local game launcher can connect to local server
+> The location and the name of the build folder is optional.
+
+Configure the engine in a non-monolithic mode with o3de-multiplayersample project.
+
+```shell
+C:\git\o3de> cmake -S . -B build\non_mono -DLY_MONOLITHIC_GAME=OFF -DLY_PROJECTS="C:\git\o3de-multiplayersample"
+```
+
+> `-DLY_MONOLITHIC_GAME=OFF` is the default value but for clarity it's specified here explicitly.
+
+Build the Editor. This will compile the project and necessary gems to produce the required game assets.
+
+```shell
+C:\git\o3de> cmake --build build\non_mono --target Editor --config profile
+```
+
+Run the Asset Processor with o3de-multiplayersample and let all the assets get processed.
+
+```shell
+C:\git\o3de> .\build\non_mono\bin\profile\AssetProcessor.exe --project-path C:\git\o3de-multiplayersample
+```
 
 ### Build AssetBuilder
 
-You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built.
+You will need to build the [AssetBundler](https://www.o3de.org/docs/user-guide/packaging/asset-bundler/overview/) tool if not built to create game bundles (.pak files).
 
 For example:
 ```shell
-cmake --build build\windows --target AssetBundler --config profile -- /m /nologo
+C:\git\o3de> cmake --build build\non_mono --target AssetBundler --config profile
```

Comment:
you could combine this with the Editor build command above by adding the `AssetBundler` target after `Editor`